src/messages/mod.rs

```diff
@@ -306,6 +310,18 @@ impl Message {
         &self.hash
     }
 
+    /// Elders will aggregate a group sig before
+    /// they all all send one copy of it each to dst.
+    pub fn aggregate_at_src(&self) -> bool {
+        matches!(self.src, SrcAuthority::Section { .. })
+    }
+
+    /// Elders will send their signed message, which
+    /// recipients aggregate.
+    pub fn aggregate_at_dst(&self) -> bool {
+        matches!(self.src, SrcAuthority::BlsShare { .. })
+    }
```

Comment:
Minor suggestion: instead of these two methods, have one which returns `Aggregation`.